css/liveblog.css

```diff
@@ -150,8 +150,8 @@ a.liveblog-form-entry::-webkit-input-placeholder {
 	margin: 1px;
 	border: solid 1px rgba(0, 0, 0, 0.0);
 	padding: 2px 1px;
-	width: 20px;
-	height: 20px;
+	width: 24px;
```

Comment:
It's because there's more vertical padding then horizontal, I basically tried to mimic `box-sizing: content-box;` to make sure the parent element would fit it's content instead of messing with background sizes or something like that. But I agree, I just reduced the padding as well so they can be made squares, I think they look better now.

<img width="255" alt="screen shot 2016-11-21 at 09 31 10" src="https://cloud.githubusercontent.com/assets/661330/20477411/6e383a28-afcd-11e6-86ce-46db91fbc6c4.png">